src/main/groovy/grails/plugins/elasticsearch/mapping/SearchableClassMappingConfigurator.groovy

```diff
@@ -114,10 +114,6 @@ class SearchableClassMappingConfigurator implements ElasticSearchConfigAware {
                 .findAll { it.isRoot() }
                 .collect { it.domainClass.fullName.toLowerCase() } as Set<String>
```

Comment:
Is there any specific reason to use domain class name as index vs property indexName of `SearchableClassMapping`?

Reply:
hm, that fix for another bug =).  Sorry for that
Old code not propertly works with hierarchy of entities. In one place it's writes to index with childe class name and at another it reads from parent name index (because of Hibernate magic) For Grials 3 that works well, but during migration to grails 4 the logic in gorm was changed. 

I have fixed it on local branch 6 months ago and fully forgot about the problem.